### PR TITLE
code review repository

### PR DESCRIPTION
**Pull request by alexisuwimana**
Review of code

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    nav {
+      position: fixed;
+      width: 100%;
+      top: 0;
+      left: 0;
+      background-color: #cc0000;
+      color: white;
+      font-family: 'Exo 2', sans-serif;
+      padding: 1em;
+    }
```
Comment (alexisuwimana) [inline styles.css:20]: hey, @alexisuwimana  great job so far but you need to improve your CSS Styles, 

- [ ]  it is better to leave an empty space at the end of the code

**Review by alexisuwimana — COMMENTED** [review]
it is better to configure linters and make sure no errors this project must not be reviewed

File: styles.css (hunk 1 above)
Comment (henatan99) [reply styles.css:20]: - [ ] Nice to use friendly words. But it lacks an explanation why this requested change is important. Plus, kindly mind grammatical and spelling errors. :pray: Please, follow [this link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) to learn more on how to give a code review.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
```markdown
Hi Alexis Uwimana, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive title and summary on PR, does not have a professional README, and linter checks are failing on PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

Invalid Code Review Request **does not count** into the code reviews limit._
```

**Comment by henatan99** [comment]
Hi @alexisuwimana , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] To properly make use of the review summary field, I suggest you write the status of the review and some summary sentences in this field. 

image

_Check the comments under your review comments._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <a href="www.google.com"></a>
+
+  </div>
+
+
+</section>
+
+<a id="profile-link" href="https://github.com/sj1978/anibal-mohamed19" target="_blank">GitHub</a>
+
+  <!--
```
Comment (alexisuwimana) [inline index.html:33]: hey, @alexisuwimana great job so far but you need to improve your Code by removing inline Styles in order to have a clean code please follow stilling rules

File: index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<script></script>
+
```
Comment (alexisuwimana) [inline index.html:4]: - [ ] great job so far but you need to remove empty tags in order to have clean code

**Review by alexisuwimana — COMMENTED** [review]
**Required Changes ♻️**

- [ ] you need to improve Linters configuration before you submit the project 
- [ ] inline styles must be removed 
- [ ] comment (unused code) must be removed in order to have a clean code

Cheers and Happy coding!👏👏👏

## **Feel free to leave any questions or comments in the PR thread if something is not 100% clear.**

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

File: index.html (hunk 2 above)
Comment (henatan99) [reply index.html:33]: This is great! :clap:

File: index.html (hunk 3 above)
Comment (henatan99) [reply index.html:4]: Nice job. Thank you for making the requested changes. :+1:

**Review by EricMbouwe — APPROVED** [review]
Hi @alexisuwimana, 

# Status: Approved✔️

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**